Fluid/Filters/ColorFilters.cs

```diff
@@ -802,9 +811,19 @@ private static double QqhToRgb(double q1, double q2, double hue)
                     );
             }
 
-            public override string ToString() => A == DefaultTransperency
```

Comment:
```csharp
public override string ToString() => A == DefaultTransperency
                ? $"rgb({R}, {G}, {B})".ToString(CultureInfo.InvariantCulture)
                : $"rgba({R}, {G}, {B}, {Math.Round(A, 1)})".ToString(CultureInfo.InvariantCulture);
```

Reply:
This does not seem to work 🤔 If I add the ToString directly to the literal, the compiler seems to infer that it is a regular string instead...

Reply:
Oh really

Reply:
If it doesn't compile which I don't  why ;) keep your previous changes

Reply:
Apparently there is [FormattableString.Invariant](https://docs.microsoft.com/en-us/dotnet/api/system.formattablestring.invariant) to do what you want to do. Check it out I pushed an updated version :).